tests/CppUTest/TestFailureTest.cpp

```diff
@@ -442,4 +442,4 @@ TEST(TestFailure, UnexpectedExceptionFailure_StandardException)
     STRCMP_CONTAINS("runtime_error", f.getMessage().asCharString());
     STRCMP_CONTAINS("' was thrown: Some error", f.getMessage().asCharString());
 }
-#endif
\ No newline at end of file
+#endif /* CPPUTEST_USE_STD_CPP_LIB */
```

Comment:
I'll merge these, but AFAIK this isn't consistently used and I would myself not prefer comments at the end of the #endif.

Reply:
No problem, I'm going to remove them :+1: